dashboard/test/controllers/api/v1/pd/workshop_organizer_survey_report_controller_test.rb

```diff
@@ -15,20 +15,20 @@ class Api::V1::Pd::WorkshopOrganizerSurveyReportControllerTest < ::ActionControl
     sign_in admin
 
     get :index, params: {course: Pd::Workshop::COURSES.first}
-    assert :success
+    assert_response :success
   end
 
   test 'organizers can view surveys' do
     sign_in @organizer
     get :index, params: {course: Pd::Workshop::COURSES.first}
-    assert :success
+    assert_response :success
   end
 
-  test 'teachers who aren\'t organizers can\'t view surveys' do
+  test "teachers who are not facilitators can't view workshops that are not their own" do
```

Comment:
Why change `surveys` to `workshops that are not their own`? The former seems more appropriate here. There is no actual workshop involved since they param is a course name and the API returns forbidden.